### ISSUE TITLE
Can UI support emoji?

### ISSUE THREAD

**Issue by chaosprint** [AmbientRun/Ambient#330]
Private Use Area like `\u{f1e2}` can be used but not the ideal emoji

**Comment by ten3roberts** [comment]
The UI supports emojis/unicode to the extent of the underlying font.

The default font is a patched Nerd Font, which means that these also work: https://www.nerdfonts.com/cheat-sheet

**Comment by chaosprint** [comment]
> The UI supports emojis/unicode to the extent of the underlying font.
> 
> The default font is a patched Nerd Font, which means that these also work: https://www.nerdfonts.com/cheat-sheet

I tried dozens of them and half cannot be rendered, displaying an empty square ☐.

**Comment by philpax** [comment]
Yep, I can also confirm this. I wonder if the same issue persists with a custom URL?

**Comment by ten3roberts** [comment]
Which ones did you use?

The recent version of nerdfonts have had to move a lot of code points around to accomodate larger icon sets. We are using a very old version of nerd fonts, so the codepoints are likely also incorrect

**Comment by philpax** [comment]
![image](https://github.com/AmbientRun/Ambient/assets/707827/bdda15fa-18d3-4d08-a3f3-efd06b65b196)

**Comment by ten3roberts** [comment]
Is thay upwards arrow and rofl glyphs?

**Comment by chaosprint** [comment]
> Which ones did you use?
> 
> The recent version of nerdfonts have had to move a lot of code points around to accomodate larger icon sets. We are using a very old version of nerd fonts, so the codepoints are likely also incorrect

I am pretty sure this one `nf-md-hand_pointing_right` won't work \u{f02c7}, I was trying to use that for killing broadcast.

**Comment by philpax** [comment]
> Is thay upwards arrow and rofl glyphs?

→ 🤣
